src/hooks/usePaginatedActions.ts

```diff
@@ -37,28 +37,35 @@ export const usePaginatedActions = (): UsePaginatedActionsReturn => {
     fetchPolicy: 'network-only',
   });
   const { items, nextToken } = data?.getActionsByColony || {};
+  const [visibleActions, setVisibleActions] = useState(ITEMS_PER_PAGE);
```

Comment:
How about `visibleActionsCount` to be specific it's a number, not an array of objects?